trieChecker/README.md

```diff
@@ -0,0 +1,7 @@
+# How to use
+
+1. compile the binary by issuing a `go build` command in elrond-tools-go/trieChecker/cmd directory
+2. create a `db` directory and place inside directories `0`, `1` ... that contains the state data
```

Comment:
the `0`, `1` paths are handy, but I think it would be easier if this tool would know to handle the exact pattern of node's databases directories (Epoch_X/Shard_x) as it would be easier to simply copy-paste the node's DB directory and then run this tool

Reply:
however, open discussions. can remain like this for now

Reply:
will not do it for now as that will complicate even more this tool: It will need the current epoch to load which translates into more directories parsing